test/conftest.py

```diff
@@ -97,17 +97,12 @@ def patch_mongomock(monkeypatch):
 
 
 @pytest.fixture
-def test_db(monkeypatch, request) -> Iterator[Datalayer]:
+def test_db() -> Iterator[Datalayer]:
     from superduperdb import CFG
     from superduperdb.base.build import build_datalayer
 
     # mongodb instead of localhost is required for CFG compatibility with docker-host
-    data_backend = 'mongodb://superduper:superduper@mongodb:27017/test_db'
-    data_backend = os.environ.get('SUPERDUPER_MONGO_URI', data_backend)
-    db_name = data_backend.split('/')[-1]
-
-    artifact_store = 'filesystem:///tmp/artifacts'
-    monkeypatch.setattr(CFG, 'artifact_store', artifact_store)
+    db_name = CFG.data_backend.split('/')[-1]
```

Comment:
When is the data_backend being populated with the actual values? 

Before or after the build_datalayer ?